src/containers/App/App.js

```diff
@@ -46,5 +46,5 @@ export default withRouter(
 
 App.propTypes = {
   getNotes: PropTypes.func,
-  notes: PropTypes.object
+  notes: PropTypes.array
```

Comment:
Fixed in reference to: https://github.com/shannonmoranetz/trapper-keeper-ui/pull/32#discussion_r257445480

Reply:
Thank you for fixing this! 👍